charts/rancher-vsphere-cpi/Chart.yaml

```diff
@@ -6,16 +6,16 @@ annotations:
   catalog.cattle.io/rancher-version: '>= 2.6.0-0 <= 2.6.99-0'
   catalog.cattle.io/release-name: vsphere-cpi
 apiVersion: v1
-appVersion: 1.0.1
+appVersion: 1.22.5
```

Comment:
`appVersion` should be 1.1.0 to match `version`. Our chart is technically based on all their branches from 1.18 to 1.22, and not 1.22.5 specifically so it would be misleading to specify 1.22.5 here.

Reply:
Made the required changes.